[DOCS-8024 to 8028] Add new Obs Pipelines component docs (#23623)
DataDog/documentation: content/en/observability_pipelines/sensitive_data_redaction/syslog.md

---
title: Sensitive Data Redaction for Syslog
kind: document
disable_toc: false
---

## Overview

Sensitive data, such as credit card numbers, bank routing numbers, and API keys, can be revealed unintentionally in your logs, which can expose your organization to financial and privacy risks.

> Use the Observability Pipelines to identify, tag, and optionally redact or hash sensitive information before routing logs to different destinations and outside of your infrastructure. You can use out-of-the-box scanning rules to detect common patterns such as email addresses, credit card numbers, API keys, authorization tokens, and more. Or, create custom scanning rules using regex patterns to match sensitive information.

Reviewer: ```suggestion
Use the Observability Pipelines to identify, tag, and optionally redact or hash sensitive information before routing logs to different destinations and outside of your infrastructure. You can use out-of-the-box scanning rules to detect common patterns such as email addresses, credit card numbers, API keys, authorization tokens, and more. You can also create custom scanning rules using regex patterns to match sensitive information.
```

> {{< img src="observability_pipelines/use_cases/sensitive_data_redaction.png" alt="The log sources, processors, and destinations available for the split logs use case" width="100%" >}}

Reviewer: ```suggestion
{{< img src="observability_pipelines/use_cases/sensitive_data_redaction.png" alt="The log sources, processors, and destinations available for sensitive data redaction" width="100%" >}}
```

This document walks you through the following steps:
1. The [prerequisites](#prerequisites) needed to set up Observability Pipelines
1. [Setting up Observability Pipelines](#set-up-observability-pipelines)
1. [Sending logs to the Observability Pipelines Worker](#send-logs-to-the-observability-pipelines-worker-over-syslog)

## Prerequisites

{{% observability_pipelines/prerequisites/syslog%}}

## Set up Observability Pipelines

1. Navigate to [Observability Pipelines][1].
1. Select the **Sensitive Data Redactions** template to create a new pipeline.

> 1. Select **Rsyslog or Syslog-ng** as the source.

Reviewer: ```suggestion
1. Select **Rsyslog** or **Syslog-ng** as the source.
```

{{< tabs >}}
{{% tab "Filter" %}}

{{% observability_pipelines/processors/filter %}}

{{% /tab %}}
{{% tab "Edit fields" %}}

{{% observability_pipelines/processors/remap %}}

{{% /tab %}}
{{% tab "Sample" %}}

{{% observability_pipelines/processors/sample %}}

{{% /tab %}}
{{% tab "Grok Parser" %}}

{{% observability_pipelines/processors/grok_parser %}}

{{% /tab %}}
{{% tab "Quota" %}}

{{% observability_pipelines/processors/quota %}}

{{% /tab %}}
{{% tab "Reduce" %}}

{{% observability_pipelines/processors/reduce %}}

{{% /tab %}}
{{% tab "Dedupe" %}}

{{% observability_pipelines/processors/dedupe %}}

{{% /tab %}}
{{% tab "Sensitive Data Scanner" %}}

{{% observability_pipelines/processors/sensitive_data_scanner %}}

{{% /tab %}}
{{% tab "Add hostname" %}}

{{% observability_pipelines/processors/add_hostname %}}

{{% /tab %}}
{{% tab "Parse JSON" %}}

{{% observability_pipelines/processors/parse_json %}}

{{% /tab %}}
{{% tab "Enrichment table" %}}

{{% observability_pipelines/processors/enrichment_table %}}

{{% /tab %}}
{{< /tabs >}}

## Install the Observability Pipelines Worker
1. Select your platform in the **Choose your installation platform** dropdown menu.

> 1. Enter the Syslog address. This is a Syslog compatible endpoint that the Worker exposes that your applications send logs to. The Observability Pipelines Worker listens on this address for incoming logs.

Reviewer: ```suggestion
1. Enter the Syslog address. This is a Syslog-compatible endpoint, exposed by the Worker, that your applications send logs to. The Observability Pipelines Worker listens on this address for incoming logs.
```
